templates/zerver/api/changelog.md

```diff
@@ -10,6 +10,10 @@ below features are supported.
 
 ## Changes in Zulip 4.0
 
+**Feature level 39**
+
+* The remove event for `realm_bot` is renamed to deactivate.
```

Comment:
Let's use an inline code block around `remove` and `deactivate`.  Also, see below for example language to borrow for explaining this.

Reply:
Made the suggested changes also added a commit fro renaming `delete` event to `change_owner`.